.scrutinizer.yml

```diff
@@ -9,27 +9,117 @@ before_commands:
 tools:
     external_code_coverage:
         timeout: 3600
-    php_code_coverage:
-        enabled: true
-    php_code_sniffer:
-        enabled: false
-    php_cpd:
-        enabled: true
-        excluded_dirs: ["bin", "docs", "tests", "tools", "vendor"]
-    php_cs_fixer:
-        enabled: false
-    php_loc:
-        enabled: true
-        excluded_dirs: ["bin", "docs", "tests", "tools", "vendor"]
-    php_mess_detector:
-        enabled: true
-        filter:
-            paths: ["lib/*"]
-    php_pdepend:
-        enabled: true
-        excluded_dirs: ["docs", "examples", "tests", "vendor"]
-    php_analyzer:
-        enabled: true
-        filter:
-            paths: ["lib/*", "tests/*"]
-    sensiolabs_security_checker: true
+
+filter:
+    excluded_paths:
+        - docs
+        - tools
+
+checks:
+    php:
+        code_rating: true
```

Comment:
The point is to also keep https://scrutinizer-ci.com/g/doctrine/doctrine2/issues/master updated with things that we consider to be improved (and which people could just pick up).

I mean, the defaults are fine but they're too generic... maybe we could find a middle ground?